xla/service/gpu/transforms/explicit_stream_annotation_async_wrapper.cc

```diff
@@ -39,12 +39,18 @@ static absl::StatusOr<bool> AsynchronizeInstruction(HloInstruction* instr) {
     return false;
   }
   HloComputation* computation = instr->parent();
+  auto original_attributes = instr->frontend_attributes();
+  // Clear the op's frontend attributes.
```

Comment:
Let's add a couple of lines of explanation why we're doing this. It's not obvious for the casual reader.